CHANGES.md

```diff
@@ -13,6 +13,8 @@ Deprecations:
 
 Language Improvements:
 
+- enh(javascript) setTimeout|setInterval now highlighted (#2514) [Vania Kucher][]
+- fix(javascript) prevent setter keyword `set` conflicting with setTimeout|setInterval (#2514) [Vania Kucher][]
```

Comment:
You need to add yourself around line 23/24 also.... so it knows where to link to.